### ISSUE TITLE
finetune training code on codellama

### ISSUE THREAD

**Issue by PoseidomWong** [nlpxucan/WizardLM#188]
Could you offer the training code on codellama? did it trained on llamaX?

**Comment by nlpxucan** [comment]
yes, it is the same with the previous wizard models, it is trained on llama-X

**Comment by PoseidomWong** [comment]
also use train_wizardcoder.py without any modify？

**Comment by mrxujun** [comment]
I encountered an error while using train_wizardcoder.py. What could be the reason for this?  Usage  CUDA_VISIBLE_DEVICES=5 deepspeed train_wizardcoder.py
